src/SqlClient/Shared.fs

```diff
@@ -92,28 +111,17 @@ type Column = {
         Description = defaultArg description ""
     }
 
-    override this.ToString() = 
-        sprintf "%s\t%s\t%b\t%i\t%b\t%b\t%b\t%s\t%s" 
-            this.Name 
-            this.TypeInfo.ClrTypeFullName 
-            this.Nullable 
-            this.MaxLength
-            this.ReadOnly
-            this.Identity
-            this.PartOfUniqueKey
-            this.DefaultConstraint
-            this.Description
-
-and TypeInfo = {
-    TypeName: string
-    Schema: string
-    SqlEngineTypeId: int
-    UserTypeId: int
-    SqlDbType: SqlDbType
-    IsFixedLength: bool 
-    ClrTypeFullName: string
-    UdttName: string 
-    TableTypeColumns: Column[] Lazy
+
+and [<DataContract;CLIMutable>] TypeInfo = {
+    [<DataMember>] TypeName       : string
+    [<DataMember>] Schema         : string
+    [<DataMember>] SqlEngineTypeId: int
+    [<DataMember>] UserTypeId     : int
+    [<DataMember>] SqlDbType      : SqlDbType
+    [<DataMember>] IsFixedLength  : bool 
+    [<DataMember>] ClrTypeFullName: string
+    [<DataMember>] UdttName       : string 
+    [<DataMember>] TableTypeColumns: Column[]
```

Comment:
Why you reformatted the code? Keep the current style.

Reply:
* compiles to same IL
* when I want to instanciate a record, I come there and copy paste all the field names in a single sweep
* with a bit more effort, it reads better


the code is going to churn, but I'd like to get feedback on exposing `.Columns` member, and probably doing the same for command parameters, the code style can be reviewed when the code settled down if we like.